### PR TITLE
[New] Add angular-growl@0.4.0 w/ git auto-update

### LINKED ISSUE #8649: [Request] Add angular-growl
**Library name:** angular-growl
**Git repository url:**https://github.com/marcorinck/angular-growl
**npm package url(optional):** 
**License(s):**
**Official homepage:**
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by sashberd**
close #8649, cc @marcorinck

Pull request for issue: #8649
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/marcorinck/angular-growl
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/angular-growl
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 25
   - Count of stars: 504
   - Count of forks: 207
 * NPM download stats (optional):
   - Downloads in the last day: 1
   - Downloads in the last week: 17
   - Downloads in the last month: 236
 
# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/angular-growl/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "name": "angular-growl",
+  "version": "0.4.0",
+  "description": "growl like notifications for angularJS projects, using bootstrap alert classes",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/marcorinck/angular-growl.git"
+  },
+  "author": {
+    "name": "Marco Rinck"
+  },
```
Comment (sufuf3) [inline ajax/libs/angular-growl/package.json]: Let's modified to `"author": "Marco Rinck",` like https://github.com/marcorinck/angular-growl/blob/master/bower.json#L2
We's like let author in one line, if the field contain only fill in author's name.

Thanks.

File: ajax/libs/angular-growl/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "license": "MIT",
+  "filename": "angular-growl.min.js",
```
Comment (sufuf3) [inline ajax/libs/angular-growl/package.json]: Let's move this line between `name` and `version` fields.
It's more useful for developers.

Thanks.

**Comment by sashberd** [comment]
@sufuf3 Thanks for feedback. The PR was updated. FYR

**Comment by sufuf3** [comment]
@extend1994 @maruilian11
Please help review this PR, Thank you.

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Comment by sashberd** [comment]
@extend1994 ping

**Comment by extend1994** [comment]
@PeterDaveHello The forked library(angular-growl-2) from this one had been merged (https://github.com/cdnjs/cdnjs/pull/12145. I found angular-growl-2 includes all versions of angular-growl. Should this library be added? Thanks.

**Comment by PeterDaveHello** [comment]
@sashberd same problem here, file newline char in v0.1.0 was broken, and, you should minify it. I'll just remove that commit this time to help save time.

**Comment by PeterDaveHello** [comment]
@sufuf3 @extend1994 the above issue(touched file format) appears many times, any idea how to prevent it?

**Pull request merged**